### DISCUSSION TITLE
tidal-dl.json

### DISCUSSION THREAD

**Discussion by enigmaoftech** [RandomNinjaAtk/arr-scripts#128] [Q&A] [answered]
I've made some adjustments to the tidal-dl.json. Will these settings stay when things update or will I need to monitor this?

I've updated audioQuality to Master instead of HiFi and multiThread to True.

**Comment by RandomNinjaAtk** [comment, marked as answer]
The settings are overwritten on every script run, its not designed to be altered.

**Reply by enigmaoftech** [reply]
Thank you, I appreciate the info. I will see if mounting that specific file will keep that static. Might be worth at least updating yours from Hifi to Master. It won't download master files without that change.
